src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -190,12 +191,25 @@ object ValidateUserOwnsFilesForFileStatusInput extends AuthorisationTag {
   }
 }
 
+object ValidateUserOwnsFilesForDeleteMetadataInput extends AuthorisationTag {
```

Comment:
There are now two validation Objects which are essentially doing the same things:

- `ValidateUserOwnsFilesForDeleteMetadataInput`
- `ValidateUserOwnsFiles`

Is it possible to use a trait similar to how the `trait UserOwnsConsignment`is used, eg:

```
trait UserOwnsFiles  {
  def fileIds: Set[UUID]
}
```
Then ensure the filed input case classes extent this trait, and then have a single validation Object: `ValidateUserOwnsFiles`

Reply:
Yeah, good point. I have used the generic type in `ValidateUserOwnsFiles` and used it in the mutations, thanks.